.github/workflows/docker_build.yml

```diff
@@ -50,17 +57,27 @@ jobs:
         uses: docker/setup-buildx-action@4fd812986e6c8c2a69e18311145f9371337f27d4 # v3.4.0
         with:
           buildkitd-flags: "--debug --allow-insecure-entitlement security.insecure --allow-insecure-entitlement network.host"
-          platforms: "linux/amd64"
```

Comment:
Just out of curiosity why ? 
i read that the v4 supports newer CPU instruction sets? 
but why do we make a choice to use it ?

Reply:
"linux/amd64/v4" ==> force usage of AVX512/AVX512fp extension set or try to virtualize it or fail
"linux/amd64"      ==> use the highest available extension set and do not enforce instruction set 

So this is a question that should be discussed and decided. Main issue that comes to my mind is fact that docker builds are run on VMs given by GitHub runners and second is that all xeons use v4 optimizations and more or less enforcing it brings  benefetis only/mostly. This of course - when used for release and push to docker registry would limit compatibility.

Long things short - I can set it back to more generic "linux/amd64"

Reply:
Nope i was just curious i know way to little about github actions to make this decisions myself I'm totally fine with this change